test/fusion_auth/plugs/authorize_jwt_test.exs

```diff
@@ -81,6 +87,22 @@ defmodule FusionAuth.Plugs.AuthorizeJWTTest do
                |> AuthorizeJWT.call()
     end
 
+    test "Succesfully refreshes the access token", %{token: token, refresh_token: refresh_token} do
```

Comment:
Should we add another test in here to ensure the verification of the JWT is working as expected since we're using JOSE now instead of FusionAuth? Or is there a test already in here to capture that?

Reply:
Also, should this test check the new token expiration date?

Reply:
The other tests in this module all already use jose for the verification of the token. I believe that checking to make sure the new token != the old token is sufficient as their wouldn't be a new token if the refresh call failed